plugins.json

```diff
@@ -62,6 +62,15 @@
       "url": "https://unpkg.com/jbrowse-plugin-ideogram/dist/jbrowse-plugin-ideogram.umd.production.min.js",
       "license": "Apache License 2.0",
       "image": "https://raw.githubusercontent.com/GMOD/jbrowse-plugin-list/main/img/ideogram-screenshot-fs8.png"
+    },
+    {
+      "name": "CIVIC",
+      "authors": ["Colin Diesh"],
+      "description": "JBrowse 2 plugin for fetching data from the CIVIC clinical interpretation of cancer variants",
+      "location": "https://github.com/cmdcolin/jbrowse-plugin-civic",
+      "url": "https://unpkg.com/jbrowse-plugin-civc/dist/jbrowse-plugin-civic.umd.production.min.js",
+      "license": "Apache License 2.0",
+      "image": "https://raw.githubusercontent.com/GMOD/jbrowse-plugin-list/main/img/civic-screenshot-fs8.png"
```

Comment:
404 when I tried to look at this image, is it there?

Reply:
it refers to main branch but image will work when merged